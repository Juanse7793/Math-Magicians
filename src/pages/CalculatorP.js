import React from 'react';
import Calculator from '../components/calculator';
import './Style.css';

const CalculatorPage = () => (
  <div className="CalculatorPage">
    <div className="CalculatorPage-title">
      <h2>Let`s Do Math</h2>
    </div>
    <div className="CalculatorPage-calculator">
      <Calculator />
    </div>
  </div>
);

export default CalculatorPage;
