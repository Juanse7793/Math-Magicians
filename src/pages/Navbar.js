import React from 'react';
import './Style.css';

const Navbar = () => (
  <div className="navbar">
    <div className="navbar-title">
      <h1>Math Magicians</h1>
    </div>
    <div className="navbar-links">
      <a className="links" href="/">Home</a>
      <a className="links" href="/calculator">Calculator</a>
      <a className="links" href="/quotes">Quotes</a>
    </div>
  </div>
);

export default Navbar;
