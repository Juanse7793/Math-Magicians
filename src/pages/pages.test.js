import renderer from 'react-test-renderer';
import Quotes from './quotes';
import CalculatorP from './calculatorP';
import Home from './home';
import Navbar from './Navbar';

describe('Pages renders correctly', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Quotes />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders correctly', () => {
    const tree = renderer.create(<CalculatorP />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders correctly', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  it('renders correctly', () => {
    const tree = renderer.create(<Navbar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
