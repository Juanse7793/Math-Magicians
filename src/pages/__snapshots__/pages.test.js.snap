// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages renders correctly renders correctly 1`] = `
<div
  className="Quotes"
>
  <div
    className="Quotes-title"
  >
    <h2>
      “A mathematician is a person who can find analogies between theorems; a better mathematician is one who can see analogies between proofs and the best mathematician can notice analogies between theories.“- Stefan Banach.
    </h2>
  </div>
</div>
`;

exports[`Pages renders correctly renders correctly 2`] = `
<div
  className="CalculatorPage"
>
  <div
    className="CalculatorPage-title"
  >
    <h2>
      Let\`s Do Math
    </h2>
  </div>
  <div
    className="CalculatorPage-calculator"
  >
    <div
      className="calculator"
    >
      <div
        className="container"
      >
        <div
          className="output"
        >
          <h2
            className="display"
          >
            0
          </h2>
        </div>
        <div
          className="numbers"
        >
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="AC"
          >
            AC
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="+/-"
          >
            +/-
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="%"
          >
            %
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
            value="÷"
          >
            ÷
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="7"
          >
            7
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="8"
          >
            8
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="9"
          >
            9
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
            value="x"
          >
            x
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="4"
          >
            4
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="5"
          >
            5
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="6"
          >
            6
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
            value="-"
          >
            -
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="1"
          >
            1
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="2"
          >
            2
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="3"
          >
            3
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
            value="+"
          >
            +
          </button>
        </div>
        <div
          className="numbers-last"
        >
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="0"
          >
            0
          </button>
          <button
            className="number"
            onClick={[Function]}
            type="button"
            value="."
          >
            .
          </button>
          <button
            className="operator"
            onClick={[Function]}
            type="button"
            value="="
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Pages renders correctly renders correctly 3`] = `
<div
  className="Home"
>
  <div
    className="Home-title"
  >
    <h2>
      Welcome to the Math Magicians Page
    </h2>
  </div>
  <div
    className="Home-p"
  >
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum accusantium repellat sunt quod reiciendis alias libero molestiae iusto voluptas deleniti quae dolore dolor, voluptate autem? Qui dolorum ab maxime adipisci.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum accusantium repellat sunt quod reiciendis alias libero molestiae iusto voluptas deleniti quae dolore dolor, voluptate autem? Qui dolorum ab maxime adipisci.
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Nostrum accusantium repellat sunt quod reiciendis alias libero molestiae iusto voluptas deleniti quae dolore dolor, voluptate autem? Qui dolorum ab maxime adipisci.
    </p>
  </div>
</div>
`;

exports[`Pages renders correctly renders correctly 4`] = `
<div
  className="navbar"
>
  <div
    className="navbar-title"
  >
    <h1>
      Math Magicians
    </h1>
  </div>
  <div
    className="navbar-links"
  >
    <a
      className="links"
      href="/"
    >
      Home
    </a>
    <a
      className="links"
      href="/calculator"
    >
      Calculator
    </a>
    <a
      className="links"
      href="/quotes"
    >
      Quotes
    </a>
  </div>
</div>
`;
